recipe/meta.yaml

```diff
@@ -1,11 +1,13 @@
-{% set version = "2020.2" %}
-{% set buildnum = "254" %}  # [not osx]
-{% set buildnum = "258" %}  # [osx]
+{% set mkl_version = "2020.4" %}
+{% set version = "2020.3" %}
```

Comment:
Why are there two versions?

Reply:
Well - there are different latest versions for MKL - 2020.4 and most of other components (including DAAL and TBB) - 2020.3.   
There is no plans to get new version within 2020 product line.

For conda-forge perspective this will be repacked as 2020.3 but overall i don't see this is large problem so far.

Fundamentally we need to have separate repos for repacking DAAL and MKL. And probably some other components in future (if we will need them)